cmd/host/main.go

```diff
@@ -129,6 +153,48 @@ func run(configuration *types.Configuration, endpoints []string) error {
 		}
 	}()
 
+	if vpnkitSocket != "" {
+		vpnkitListener, err := transport.Listen(vpnkitSocket)
+		if err != nil {
+			return err
+		}
+		go func() {
+			for {
+				conn, err := vpnkitListener.Accept()
+				if err != nil {
+					log.Errorf("vpnkit accept error: %s", err)
+					continue
+				}
+				go func() {
+					if err := vn.AcceptVpnKit(conn); err != nil {
+						log.Errorf("vpnkit error: %s", err)
+					}
+				}()
+			}
+		}()
+	}
+
+	if qemuSocket != "" {
+		qemuListener, err := transport.Listen(qemuSocket)
+		if err != nil {
+			return err
+		}
+		go func() {
+			for {
+				conn, err := qemuListener.Accept()
+				if err != nil {
+					log.Errorf("qemu accept error: %s", err)
+					continue
+				}
+				go func() {
+					if err := vn.AcceptQemu(conn); err != nil {
+						log.Errorf("qemu error: %s", err)
+					}
+				}()
+			}
+		}()
+	}
+
 	ln, err := vn.Listen("tcp", fmt.Sprintf("%s:8080", configuration.GatewayIP))
```

Comment:
long term I would like to see this port value configurable (i'll do a pr for it).  Port 8080 is frequently used by containers and therefor port conflicts are quite likely.  Ok for now.